src/server/src/main/java/io/cassandrareaper/service/RepairRunner.java

```diff
@@ -82,17 +82,14 @@ private RepairRunner(
     this.repairRunId = repairRunId;
     Optional<RepairRun> repairRun = context.storage.getRepairRun(repairRunId);
     assert repairRun.isPresent() : "No RepairRun with ID " + repairRunId + " found from storage";
-    cluster = context.storage.getCluster(repairRun.get().getClusterName());
-    assert cluster.isPresent() : "No Cluster with name " + repairRun.get().getClusterName() + " found from storage";
+    this.cluster = context.storage.getCluster(repairRun.get().getClusterName());
     RepairUnit repairUnitOpt = context.storage.getRepairUnit(repairRun.get().getRepairUnitId());
-    this.clusterName = cluster.get().getName();
-
-    Preconditions.checkArgument(cluster.isPresent(), "Cluster couldn't be found in storage");
+    this.clusterName = cluster.getName();
     String keyspace = repairUnitOpt.getKeyspaceName();
-    int parallelRepairs
-        = getPossibleParallelRepairsCount(
-            clusterFacade.getRangeToEndpointMap(cluster.get(), keyspace),
-            clusterFacade.getEndpointToHostId(cluster.get()));
+
+    int parallelRepairs = getPossibleParallelRepairsCount(
```

Comment:
hem... weren't you the one asking to put the operator on the next line so it carries?
Readability of the code is better if the method call starts on the next line, but I won't be dogmatic about this.

Reply:
The rules for wrapping is to apply one (and only one) of the following approaches:
 - carry the operator,
 - carry the method calls, 
 - carry every argument.

In this example the arguments are getting carried.